plugins/outputs/redistimeseries/redists.go

```diff
@@ -0,0 +1,82 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  addr = "127.0.0.1:6379"
+
+  ## password to login Redis
+  # password = ""
+`
+
+type RedisTimeSeries struct {
+	addr     string
+	password string
+	client   *redis.Client
+}
+
+func (i *RedisTimeSeries) Connect() error {
+
+	client := redis.NewClient(&redis.Options{
+		Addr:     i.addr,
+		Password: i.password,
+		DB:       0, // use default DB
+	})
+
+	err := client.Ping().Err()
+	i.client = client
+	return err
+}
+
+func (i *RedisTimeSeries) Close() error {
+	return i.client.Close()
+}
+
+func (i *RedisTimeSeries) Description() string {
+	return "Configuration for sending metrics to RedisTimeSeries"
+}
+
+func (i *RedisTimeSeries) SampleConfig() string {
+	return sampleConfig
+}
+func (i *RedisTimeSeries) Write(metrics []telegraf.Metric) error {
+
+	if len(metrics) == 0 {
+		return nil
+	}
+
+	for _, m := range metrics {
+		now := m.Time().UnixNano() / 1000000000
+		//		tags := m.Tags() TODO add support for tags
```

Comment:
Please add tags. It shouldn't be too hard if I understand the documentation correctly
```golang
tags := make([]string, 0, len(m.TagList())
for _, tag := range m.TagList() {
    tags = append(tags, tag.Key)
    tags = append(tags, tag.Value)
}
```
and then add `"LABELS", tags...` in the `client.Do()` below if there are any tags.